x-pack/plugins/apm/public/components/app/TransactionDetails/Transaction/WaterfallContainer/Waterfall/SpanFlyout/index.tsx

```diff
@@ -73,32 +74,34 @@ export function SpanFlyout({
   const dbContext = span.context.db;
 
   return (
-    <EuiFlyout onClose={onClose} size="l">
-      <EuiFlyoutHeader hasBorder>
-        <EuiFlexGroup>
-          <EuiFlexItem grow={false}>
-            <EuiTitle>
-              <h2>Span details</h2>
-            </EuiTitle>
-          </EuiFlexItem>
+    <EuiPortal>
+      <EuiFlyout onClose={onClose} size="m" ownFocus={true}>
```

Comment:
`ownFocus` should be enough (omit `={true}`. It used to give a warning in eslint - maybe tslint is not as strict? 🤔

Reply:
Btw. is the `EuiPortal` required for the `ownFocus` stuff?

Reply:
@sqren should be enough for what?

Reply:
Should be enough for it to work, and is consistent with the rest of the codebase :)
```js
<EuiFlyout onClose={onClose} size="m" ownFocus>
```

We are currently using the default setting:
https://github.com/sqren/kibana/blob/001d22f885614c19ebe9c020d7a1e7b177b1d6f5/.eslintrc.js#L372

ESLint rule:  
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-boolean-value.md


Again, just a minor nit. I think the more important thing is to ensure that the lint rules for eslint/tslint are more aligned.

Reply:
Note to self: add this to our eslint setup https://github.com/palantir/tslint-react

Reply:
Oh I see what you're saying now. IMO we should change the lint rules to force `={true}` because as I understand it, React wants to eventually make bare attributes work the same way object shorthand works, like `const ownFocus = true; const a = { ownFocus };` so I have just gotten in the habit of being explicit with the boolean value. But yeah, that seems auto-fixable tbh.